ac/ac-train/src/Dashboards/MeanPerTryForEachInterface.jsx

```diff
@@ -51,7 +51,9 @@ const MeanPerTryForEachInterface = props => {
     }));
 
     const domain =
-      whichDash === 'error' ? { x: [0, 4], y: [0, 1] } : { x: [0, 4] };
+      whichDash === 'error'
+        ? { x: [1, count[interfaces[0]].length], y: [0, 1] }
```

Comment:
@jkolsen we can avoid this by using `iterationPerInterface`

Reply:
fixed!